presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergHiveMetadata.java

```diff
@@ -500,9 +506,16 @@ public TableStatisticsMetadata getStatisticsCollectionMetadata(ConnectorSession
     {
         Set<ColumnStatisticMetadata> columnStatistics = tableMetadata.getColumns().stream()
                 .filter(column -> !column.isHidden())
-                .flatMap(meta -> metastore.getSupportedColumnStatistics(getMetastoreContext(session), meta.getType())
-                        .stream()
-                        .map(statType -> statType.getColumnStatisticMetadata(meta.getName())))
+                .flatMap(meta -> {
+                    try {
+                        return metastore.getSupportedColumnStatistics(getMetastoreContext(session), meta.getType())
+                                .stream()
+                                .map(statType -> statType.getColumnStatisticMetadata(meta.getName()));
+                    }
+                    catch (IllegalArgumentException e) {
```

Comment:
Under what conditions is this exception thrown ?

Reply:
In `ThriftHiveMetastore#getSupportedColumnStatistics`: https://github.com/prestodb/presto/blob/244d203952c4a283891ac6c817a8f4a732aa89c4/presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastore.java#L416

Since the set of column statistics in Iceberg could differ from Hive, I didn't want to fail the operation if one of the types isn't supported by Hive.